### ISSUE TITLE
Configure property as readonly for specific user groups

### ISSUE THREAD

**Issue by bjarnef** [LottePitcher/umbraco-admin-only-property#13]
Now that most property editors support `readonly` in core, could we configure the property editor (or a new property editor) to make property editor visible to all content editors, but readonly for specific use groups (or readonly to all except the one selected/allowed on the configuration).

E.g. https://github.com/umbraco/Umbraco-CMS/blob/b69afe81f3f6fcd37480b3b0295a62af44ede245/src/Umbraco.Web.UI.Client/src/common/directives/components/property/umbpropertyeditor.directive.js#L70-L72

**Comment by LottePitcher** [comment]
Interesting idea @bjarnef! At the moment the package is just a 'wrapper' for other data types, so no custom html/js is needed. I really like this approach as means it should, in theory, work with any data type! So I wonder if there is a way to implement this state of a property without needing to add custom js.  I haven't really looked at the `readonly` support in core so sounds like I should. I'll let you know if I get anywhere!

**Comment by bjarnef** [comment]
Yeah, not sure if it is possible to set `readonly` from service side and I am not totally sure how it is set in core, e.g. for Content Picker / MNTP.
https://github.com/umbraco/Umbraco-CMS/blob/b69afe81f3f6fcd37480b3b0295a62af44ede245/src/Umbraco.Web.UI.Client/src/views/propertyeditors/contentpicker/contentpicker.html#L28

Alternatively a config option could maybe be used to set/replace view to `readonlyvalue`
https://github.com/LottePitcher/umbraco-admin-only-property/blob/4a3f4fc51c6506e688a623f0c5de460a732a030f/src/AdminOnlyProperty/AdminOnlyPropertyDataEditor.cs#L16

but may not work for all property editors as it just render the raw value, but I guess it could we useful to input properties like textbox, textarea, numeric input, tags etc.
https://github.com/umbraco/Umbraco-CMS/blob/v11/contrib/src/Umbraco.Web.UI.Client/src/views/propertyeditors/readonlyvalue/readonlyvalue.html
